docs/sphinx/user_guide/build.rst

```diff
@@ -0,0 +1,212 @@
+
+.. _build_build-label:
+
+==================
+Build and Install
+==================
+
+The build and install process for the RAJA Performance Suite is similar to
+the process for RAJA, which is described in `RAJA Build and Install <https://raja.readthedocs.io/en/develop/sphinx/user_guide/build.html#build-and-install>`_.
+
+When building the RAJA Performance Suite, RAJA and the RAJA Performance Suite 
+are built together using the same CMake configuration. The basic process for 
+generating a build space and configuration is to create a build directory and 
+run CMake in it. For example::
+
+  $ pwd
+  path/to/RAJAPerf
+  $ mkdir my-build
+  $ cd my-build
+  $ cmake <cmake args> ..
+  $ make -j
+
+For convenience and informational purposes, we maintain scripts in the 
+``scripts`` directory for various build configurations. These scripts invoke 
+associated *host-config* files (CMake cache files) in the RAJA submodule. For 
+example, the ``scripts/lc-builds`` directory contains scripts that we use 
+during development to generate build configurations for machines
+in the Livermore Computing Center at Lawrence Livermore National Laboratory. 
+These scripts are designed to be run in the top-level RAJAPerf directory. Each 
+script creates a descriptively-named build space directory and runs CMake with 
+a configuration appropriate for the platform and specified compiler(s). To 
+compile the code after CMake completes, enter the build directory and type 
+``make`` (or ``make -j <N>`` or ``make -j`` for a parallel build using N 
+processor cores, or all available processor cores on a node, respectively). 
+For example::
+
+  $ ./scripts/lc-builds/blueos_nvcc_clang.sh 10.2.89 70 10.0.1
+  $ cd build_blueos_nvcc10.2.89-sm_70-clang10.0.1
+  $ make -j 
+
+will build the code for CPU-GPU execution using the clang 10.0.1 compiler for
+the CPU and CUDA 10.2.89 for the GPU. The GPU executable code will target
+the CUDA compute architecture ``sm_70``.
+
+.. note:: The scripts in the ``scripts/lc-builds`` directory contain
+          helpful examples of running CMake to generate a variety of 
+          build configurations.
+
+When no CMake test options are provided, only the RAJA Performance Suite code 
+will be built. If you want to build both the Suite tests and RAJA tests (to
+verify that everything is built properly), pass the following options to 
+CMake: ``-DENABLE_TESTS=On`` and ``-DRAJA_PERFSUITE_ENABLE_TESTS=On``. This 
+can be done on the command line if you run CMake directly or by editing the 
+build script you are using. If you want to build the Suite tests, but not 
+RAJA tests, pass the two CMake options above plus the option 
+``-DRAJA_ENABLE_TESTS=Off``. In any case, after the build completes, you can 
+type ``make test`` to run the tests you have built and see the results.
+
+.. note:: Which kernel variants that can be run depends on which programming
```

Comment:
```suggestion
.. note:: The kernel variants that can be run depends on which programming
```

Reply:
I'll push changes based on your review comments when you are done with each file.